### PR TITLE
potench/upgrade cmp 0.0.1

### PR DESCRIPTION
**Pull request by potench**
## Background

 - [x] Upgrade CMP from upstream appnexus/cmp master 

## Test Plan

```
yarn test
```

### CODE REVIEW

File: src/docs/components/docs/configuration.jsx [hunk 1]
```diff
@@ -81,6 +81,19 @@ export default class CmpApi extends Component {
 								by pubvendors.json.
 							</span>
 						</span>
+						<span class={style.argument}>
```
Comment (danielrussellLA) [inline src/docs/components/docs/configuration.jsx:84]: why is the spacing so large in these files?

**Review by danielrussellLA — APPROVED** [review]

File: src/docs/components/docs/configuration.jsx (hunk 1 above)
Comment (danielrussellLA) [reply src/docs/components/docs/configuration.jsx:84]: nvm it's an upstream merge i see

Comment (potench) [reply src/docs/components/docs/configuration.jsx:84]: project is using `tabs` I guess this is how github renders the tabs, indentation looks right on my IDE though

Comment (potench) [reply src/docs/components/docs/configuration.jsx:84]: also, even if it's not conforming to our standard, i'd leave this as is so it's easier to manage upstream changes in the cmp (trying to touch as little of the core of the project as possible)

Comment (danielrussellLA) [reply src/docs/components/docs/configuration.jsx:84]: 👍

**Pull request merged**